src/components/Home/Hero/BetterTogether/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import HeroContainer from '../../../HeroContainer'
+import { cn } from '../../../../utils'
+
+const BetterTogether = () => {
+  return (
+    <HeroContainer className="py-14 bg-gray-hover text-light">
```

Comment:
Let's change this to dark text on light background so that the separation between the "Get Started with DVCX" and "Get Started with DVC" sections is not so stark.

This is probably wrong but something along these lines?

```suggestion
    <HeroContainer className="py-14 bg-gray-100">
```

Reply:
https://github.com/iterative/dvc.org/pull/4800